containerz/containerz.proto

```diff
@@ -258,6 +258,10 @@ message StopRequest {
 
   // If force is set, the target should attempt to forcibly kill the container.
   bool force = 2;
+
+  // If auto_restart is set, the target should start the container immediately
+  // after stopping it.
+  bool auto_restart = 3;
```

Comment:
bool restart

i think auto_restart makes it feel like the container will always restart

Reply:
Make sense. Thx for making the change!